README.md

```diff
@@ -6,79 +6,104 @@
 > [!IMPORTANT]
 > We're making some changes to this repository and we've paused adding new chains for now. We'll reopen this process once the repository is ready. The Superchain itself, of course, remains open for business.
 
-The Superchain Registry repository hosts Superchain-configuration data in a minimal human-readable form.
-
-This includes mainnet and testnet Superchain targets, and their respective member chains.
+The Superchain Registry is an index of chains which serves as the source of truth for who’s in the Superchain and what modifications they’ve made to their chains. 
+It hosts Superchain-configuration data in a minimal human-readable form and includes mainnet and testnet Superchain targets, along with their respective member chains.
 
 Other configuration, such as contract-permissions and `SystemConfig` parameters are hosted and governed onchain.
 
-A list of chains in the registry can be seen in the top level [`chainList.toml`](./chainList.toml) and [`chainList.json`](./chainList.json) files.
+A list of chains in the registry can be seen in the top level [`chainList.toml`](./chainList.toml) and [`chainList.json`](./chainList.json) files. 
```

Comment:
Do we intend to keep both of these files going forward? Seems like it would be better to pick one and deduplicate the data

Reply:
I think we decided to use both for now.